src/script/sign.cpp

```diff
@@ -60,13 +60,21 @@ static bool SignN(const vector<valtype>& multisigdata, const BaseSignatureCreato
 {
     int nSigned = 0;
     int nRequired = multisigdata.front()[0];
+    int nKeysCount = multisigdata.size() - 2;
+    MultiSigHint hint(nKeysCount, (1 << nKeysCount) - 1);
+    CScript scriptSig;
     for (unsigned int i = 1; i < multisigdata.size()-1 && nSigned < nRequired; i++)
     {
         const valtype& pubkey = multisigdata[i];
         CKeyID keyID = CPubKey(pubkey).GetID();
-        if (Sign1(keyID, creator, scriptCode, scriptSigRet))
+        if (Sign1(keyID, creator, scriptCode, scriptSig))
+        {
```

Comment:
Style: `{` on same line as `if`.

Reply:
Thanks, but the style rules for Tradecraft are pragmatically somewhat different, driven by the two needs to match the surrounding style when making targeted changes, and to minimize the possibility of introducing undetected bugs in rebasing. Expect some purposeful deviation from whatever the upstream style guide is at the time.